dashboards/api.json

```diff
@@ -655,4 +655,4 @@
   "title": "API",
   "uid": "api",
   "version": 5
-}
\ No newline at end of file
+}
```

Comment:
I'm gonna assume we don't really care about the trailing newlines, right? 😄

Reply:
yea the linter was just complaining about them